server/src/main/java/org/apache/kafka/server/config/Defaults.java

```diff
@@ -46,6 +42,10 @@ public class Defaults {
     public static final int METADATA_MAX_IDLE_INTERVAL_MS = 500;
     public static final int METADATA_MAX_RETENTION_BYTES = 100 * 1024 * 1024;
     public static final boolean DELETE_TOPIC_ENABLE = true;
+    public static final int REQUEST_TIMEOUT_MS = 30000;
+    public static final long CONNECTION_SETUP_TIMEOUT_MS = CommonClientConfigs.DEFAULT_SOCKET_CONNECTION_SETUP_TIMEOUT_MS;
```

Comment:
Do we need `CONNECTION_SETUP_TIMEOUT_MS` as it is used by `KafkaConfig.scala` only.

Reply:
These will move soon in the final PR couple of prs for moving KafkaConfig out of core. So I would keep it for now until I fully finish this jira.